ansible/roles/products/common/local-zero-force-mode-launch/tasks/main.yml

```diff
@@ -34,13 +34,23 @@
     bimanual_hands_picture: hand_extra_lite_bimanual.jpg
   when: product=='hand_extra_lite'
 
+- name: set product variable
+  set_fact:
+    hand_type_value: "hand_type:=hand_g"
+  when: '"hand_extra_lite" in product or "hand_lite" in product'
+
+- name: set product variable
+  set_fact:
+    hand_type_value: ""
+  when: 'product is not defined or "hand_e" in product'
```

Comment:
1) hand_type_value should be hand_type:=hand_e if hand_e in product (try not making code depending on defaults from launch files).
2) what's the use case of a oneliner when the product is not defined?

Reply:
1 - Sure thing, I'll add that now.
2 - It would use this, as we don't define product for teleop, and it only works with hand_e.

Reply:
2.5 - I should also add the validation stage will fail this if the product is not recognised, or is not defined for docker_deploy and server_and_nuc

Reply:
I mean, if the product is none will the whole playbook fail, or is the product not defined in the simulation-only scenario?

Reply:
So the product is always needed for docker_deploy and sever_and_nuc, but is not needed for teleop_deploy.

I just added a default for server_and_nuc and docker_deploy so its always defauled to hand-e, so we won't ever have the situation when this value is None. this product is not defined is just for teleop_deploy as we currently only have hand_e support for teleop.